mythril/analysis/modules/deprecated_ops.py

```diff
@@ -17,8 +17,9 @@ def _analyze_state(state):
 
     if instruction["opcode"] == "ORIGIN":
         log.debug("ORIGIN in function " + node.function_name)
-        title = "Use of tx.origin"
-        description = (
+        title = "Use of tx.origin is deprecated."
+        description_head = "Use of tx.origin is deprecated."
+        description_tail = (
             "The function `{}` retrieves the transaction origin (tx.origin) using the ORIGIN opcode. "
```

Comment:
The function name was removed from the description elsewhere (at least in predictable vars). Why was it removed there and not removed here?

Reply:
Sloppiness 😅